### ISSUE TITLE
ERROR Module home_screen

### ISSUE THREAD

**Issue by gadgetbec** [ColoradoSchoolOfMines/interface_sdk#61]
[home_screen-0.0.1.jar](https://github.com/ColoradoSchoolOfMines/home_screen_module/wiki/Downloads) wont run.
I'm trying to run the home_screen through the Command Line Launcher and I get this Error message:

HIIII
[2018-03-07 08:01:58,696 loaders.ModuleLoader]
 >>DEBUG Trying to set class loader for jar file: modules/home_screen-0.0.1-jar-with-dependencies.jar

[2018-03-07 08:01:58,697 loaders.ModuleLoader]
 >>DEBUG The above line was not thrown

[2018-03-07 08:01:59,972 module_management.ModuleManager]
 >>DEBUG Sending this stuff to hw instance: {handtracking=REQUIRED}

[2018-03-07 08:02:00,040 module_management.ModuleManager]
 >>ERROR Module [edu.mines.acmX.exhibit.modules.home_screen] depends on unavailable functionality

I've followed this [Tutorial](https://github.com/ColoradoSchoolOfMines/interface_sdk/wiki/Tutorial:-The-Undisputed-Step-by-Step-Guide-to-Developing-with-the-Interface-SDK) using interface SDK with dependencies [Version 0.1.1](https://s3-us-west-2.amazonaws.com/acmx.mines.edu/release/edu/mines/acmX/exhibit/interfacesdk/0.1.1/interfacesdk-0.1.1-jar-with-dependencies.jar)

**Comment by jackrosenthal** [comment]
I think all of the previous maintainers of this project have graduated and nobody is around anymore that knows this codebase... perhaps do you want to take over the project?

**Comment by adiviness** [comment]
Yes, we all graduated a few years ago, I don't believe this code has had any serious maintainers for about four years. Unfortunately I don't have any of the hardware we used either so I'm not sure how much of a help I could be in debugging launch issues.

**Comment by muff1nman** [comment]
I actually bought an old kinect when you could get them - @adiviness  if you want I could ship you my kinect if you wanted to play around with it :P I think the biggest problem is the availability of the OpenNI drivers and what not. With the apple acquisition all of the links were taken down :( Might be able to scavenge around the internet for them.

As far as the error goes it looks like its not recognizing your kinect. I would make sure to fire up some of the OpenNI demos to make sure it works before starting home screen.
